pymeasure/adapters/visa.py

```diff
@@ -138,9 +138,13 @@ def close(self):
         """
         super().close()
         try:
-            self.manager.close()
+            if self.manager.visalib.library_path == "unset":
```

Comment:
Is that the "proper" way to detect pyvisa-sim? I guess there's no defined API for the visalib type?

Reply:
pyvisa-sim internal uses the same code. I could not identify something better.